### DISCUSSION TITLE
Proper way to store a Date type in a predicate

### DISCUSSION THREAD

**Discussion by rmgraves** [fluree/db#485] [Q&A]
Hi,

New Fluree dev here! I'm looking for the proper way to represent a Date/Time in a collection predicate. I don't see any types that I can use to store it natively as a date.  I noticed that there is the "instant" predicate type, but that requires having to store the value as a 64-bit timestamp from the unix epoch.

Would it be best to store a date/time as a string type and then parse it (using .NET) to a date object? 

Thanks!
Ryan

**Comment by rmgraves** [comment]
Noticed a bunch of the previous discussions went unanswered. Any input @cap10morgan @flyingmachine @zonotope  ?

**Comment by cap10morgan** [comment]
Hi @rmgraves! I'd be happy to help. What version of Fluree are you using?

**Comment by rmgraves** [comment]
Great, thanks! 

I'm using fluree/ledger:latest from docker hub. The admin UI says it is ledger version 2.0.0?

**Comment by cap10morgan** [comment]
@rmgraves Thanks for that info!

Is there a reason the `instant` predicate type doesn't work for your needs? Most programming language date/time types have built-in methods to convert to and from that format.

**Reply by rmgraves** [reply]
It is just a format I'm not used to working with before. I have no issues with using it though. Googling conversions I can see how to do it now. 

So as an example of adding the data to the collection it would be something like this?

```
[{
    "_id": "myCollection",
    "date": 1684436690
}]
```
